README.md

```diff
@@ -182,6 +182,16 @@ For example: `0001-create-users` or `0001-modify-users`
 
 In `/src/databases/seeds` there is seed or starter data. The contents can be run manually to help developers get data in their databases. The scripts are not referenced by the software and are included for convenience. The migrations must run prior to using seed scripts.
 
+We use [SQLFluff](https://docs.sqlfluff.com/en/stable/index.html)
+to lint our SQL files. See
+[Getting Started with SQLFluff](https://docs.sqlfluff.com/en/stable/gettingstarted.html)
+for instructions on setting it up.
```

Comment:
I kind of punted on dev documentation; I know that some people like to use Docker, and some do not, and either way I'm not sure that we should proscribe a way for devs to do their work.

That said, in case it is convenient, the way I am using SQLFluff is with [`venv`](https://docs.python.org/3/library/venv.html) to avoid changing anything outside of this directory:

```sh
python3 -m venv venv
. venv/bin/activate
pip install sqlfluff
npm run lint:sqlfluff
```

Reply:
Does this mean that every time I develop for this project I'd have to active venv (assuming I run lint as part of my pre-push process).  If so: I don't love that as opposed to being able to continue to just type `npm run lint` and have it just work.

I think it would be nice to be more opinionated in the name of dev convenience (specifically lowering the amount of recurring CLI setup that would need to be done). I think it's reasonable to have a `python3` requirement as part of the dev setup instructions since we have introduced it as a language requirement.

If `python3` is a documented dev requirement, could we have the npm script run `venv` as part of the `lint:sqlfluff` command, and have npm run `pip install` as part of an `npm install` prehook (and a pipfile that locks a version of sqlfluff)

Alternatively, if we think docker is a cleaner dev dependency than python3 we could update `lint:sqlfluff` to use docker.

Reply:
I'm happy to look into that, but can that be done as part of a later PR? I'd like to get this merged sooner rather than later so that it'll start running in CI.

Reply:
@jasonaowen that's absolutely fine by me, good point!